rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2600,7 +2600,9 @@ def test_latest_upgrade_correctness(user_data_dir):
     assert tables_after_creation - tables_after_upgrade == set()
     assert views_after_creation - views_after_upgrade == set()
     new_tables = tables_after_upgrade - tables_before
-    assert new_tables == set()
+    assert new_tables == {
+        'cowswap_orders',
+    }
```

Comment:
Why split something that easily fits in 1 line to 3 lines?
```suggestion
    assert new_tables == {'cowswap_orders'}
```